include/dxc/DXIL/DxilModule.h

```diff
@@ -381,24 +395,6 @@ class DxilModule {
   template<typename T> unsigned AddResource(std::vector<std::unique_ptr<T> > &Vec, std::unique_ptr<T> pRes);
   void LoadDxilSignature(const llvm::MDTuple *pSigTuple, DxilSignature &Sig, bool bInput);
 
-  // properties from HLModule preserved as ShaderFlags
-  bool m_bDisableOptimizations;
-  bool m_bUseMinPrecision;
-  bool m_bAllResourcesBound;
-  bool m_bResMayAlias;
-
-  // properties from HLModule that should not make it to the final DXIL
-  uint32_t m_IntermediateFlags;
-  uint32_t m_AutoBindingSpace;
-
-  // porperties infered from the DXILTypeSystem
-  bool m_bHasPayloadQualifiers;
-
-  std::unique_ptr<DxilSubobjects> m_pSubobjects;
-
-  // m_bMetadataErrors is true if non-fatal metadata errors were encountered.
-  // Validator will fail in this case, but should not block module load.
-  bool m_bMetadataErrors;
```

Comment:
Well, the way the diff looks now, it seems like I moved these variables above the methods and initialized them. That's accurate enough I guess, but the way I see it, I moved the methods below the variables. The important thing to note is that there were no changes to any of the relocated code apart from giving them their default assignments here and the one removed member mentioned above.